modules/howtos/examples/views.js

```diff
@@ -19,12 +22,45 @@ async function go() {
   // end::range-query[]
 
   // tag::key-query[]
-  var result = bucket.viewQuery('landmarks', 'by_name', {
-    key: 'statue-of-liberty',
+  var result = await bucket.viewQuery('landmarks', 'by_name', {
+    key: 'landmark_10019',
   })
-
   // end::key-query[]
 }
+
+async function createViews(bucket) {
```

Comment:
This also doesn't look like a minor fix.
How did this work before?

Reply:
It just ran happily but didn't actually work it seems (the views referenced don't exist). I created the views so the code can execute happily, but didn't want to spend too much time on this as supposedly Views will go away 🤞 .